modules/developer_manual/pages/webdav_api/search/search_files.adoc

```diff
@@ -7,7 +7,14 @@
 :method: REPORT
 :content_type: text/xml
 
-The search-files report allows for searching the available files in an ownCloud user's filesystem, based on a rudimentary filename pattern match.
+The `search-files` report search through the available files in an ownCloud user's filesystem, based on a rudimentary filename pattern match.
+
+By default, the report uses the default search provider to power the search functionality. 
```

Comment:
``uses the default`` --> ``uses ownCloud's default``
Should be the same as the sentence on line 14